### ISSUE TITLE
Header Tools - contrast problem

### ISSUE THREAD

**Issue by woowoowoo** [unimelb/pattern-lib#1241]
Any light coloured background behind the header tools (search/hamburger), causes contrast problems. This is particularly apparent for the current Open Day campaign image, with a white background.

## Expected behaviour
To be able to see the header tools at all times with sufficient WCAG2-AA contrast.

## Current behaviour
a) On the uni home page, the header tools are on their dark blue backgrounds, but on :hover, they vanish. This behaviour is also seen on pattern-lib-unimelb.netlify.com.
b) On the [arts home page](https://arts.unimelb.edu.au/), which uses the MCE Home page template, there is not even a blue background., so the controls are only visible on :hover. Not sure why the MCE home page template loses the blue backgrounds.

## Possible solution
The component needs to be more resilient to cope with varying backgrounds, especially when adjusting the window size might see the elements against different levels of contrast. At all times there needs to be minimum WCAG2-AA contrast.

**Comment by jaredh-um** [comment]
@gepifanio proposed change for Gen 2 header tools - change existing `background-color` and `color` for the hover state to:

`.link-icon--vertical:hover {
background: #fff;
color: #094183;
}`

Change should look like the search button below when hovered:

![Screen Shot 2019-08-14 at 1 34 23 pm](https://user-images.githubusercontent.com/34703139/62992834-5f9de300-be98-11e9-83cc-19a0b65e1c0e.png)

I am not sure if this class is used for any other button type - it might be worth checking just in case.

**Comment by gepifanio** [comment]
PR: https://github.com/unimelb/pattern-lib/pull/1243

**Comment by dkevey** [comment]
Hi all, just to clarify that this is an issue with Gen 1 CSS - any changes need to be made to DDS system as used here: https://web.unimelb.edu.au and we would need to add a new home page template into the Gen 1 CMS.

**Comment by jaredh-um** [comment]
@dkevey to fix the example on arts.unimelb using an override (need to change background from transparent)

```
.uomcontent .page-header.floating .page-header-tools a.page-header-icon {
  background-color: #094183;
}
```

![Screen Shot 2019-08-15 at 9 08 05 am](https://user-images.githubusercontent.com/34703139/63062520-9fb6a180-bf3c-11e9-90e8-3813f89bd585.png)

**Comment by TimMansell** [comment]
Fixed in v4.15.0
